### DISCUSSION TITLE
Camera Strategy To Follow Actor But Not Go Beyond Map Bounds

### DISCUSSION THREAD

**Discussion by brettkoz** [excaliburjs/Excalibur#3204] [Q&A]
Is this possible? I'm currently using the FitScreenAndFill, with a 342x192 resolution on the game, and my maps are different sizes... I'd love for the camera to follow the actor until it gets to the bounds, at which point the camera stops and allows the actor to not be in the center.

**Comment by brettkoz** [comment]
Alright, by digging through some of the samples and showcase games I was able to find an answer to my question. You can use multiple map strategies at once. First declare the lockToActor scheme and then set up a bounding box for the map bounds and implement the limitCameraBounds strategy.

**Reply by eonarheim** [reply]
@brettkoz Catching up on my notifications!

Totally! Multiple strategies is a way to do it, the last one added has the final say (a sort of priority)
